paper_figures/code/maxsavings_contour.py

```diff
@@ -280,31 +302,40 @@ def generate_figure(df, region, month, overlay_points=[], overlay=True, save=Fal
     [0.25, 0.99]
 ])
 
+figpath = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
 # to run the analysis 
-# for region in regions:
-#     for month in months:
-#         # generate the data
-#         df = generate_data(region, month)
-
-#         # generate the figure
-#         generate_figure(
-#             df, 
-#             region, 
-#             month, 
-#             overlay_points=overlay_points, 
-#             overlay=overlay
-#         )
+for region in regions:
```

Comment:
Do we want to comment this back out once we save the data in CSV format (so that it isn't running by default). Actually, it may be better to just put a flag at the top `GENERATE_DATA = False` so we can just update one line (and of course that could be a command line arg if we really want to streamline the process)